### ISSUE TITLE
add fine cuts to bldice

### ISSUE THREAD

**Issue by jeenriquez** [UCBerkeleySETI/blimpy#71]
**Is your feature request related to a problem? Please describe.**
Not a problem, but a limitation.
Currently bldice cuts to the nearest coarse channel end.  This makes for us, but maybe not for the rest of the world. 

**Describe the solution you'd like**
Having an option to cut at any frequency maybe useful for some.

**Comment by mattlebofsky** [comment]
Maybe this is obvious, but I suggest that it cuts to the nearest coarse channel (inclusive) by default, which is its current behavior, and any finer cuts are overridden by a command line option. We don't want current pipelines to break due to new, unexpected behavior.

**Comment by jeenriquez** [comment]
yep, this is the way to go. coarse channel granularity should be default.
